lib/ember-cli-rails.rb

```diff
@@ -1,4 +1,4 @@
-require "ember-cli/railtie" if defined?(Rails)
+require "ember-cli/engine" if defined?(Rails)
```

Comment:
`Rails` constant is defined in `railties`. So in this particular case we're depending on either `rails` or `railties`. I don't really have a hard preference on either of these, but one of them definitely should be there.